test/routes/issueRoutesTest.js

```diff
@@ -0,0 +1,317 @@
+const chai = require('chai');
+const expect = require('chai').expect;
+const mongoose = require('mongoose');
+
+chai.use(require('chai-http'));
+
+const Issue = require('../../models/issueModel');
+const path = require('path');
+const File = require(path.resolve('./models/fileModel'));
+
+const testRoutes = require('../../routes/issueRoutes');
+
+describe('Issue requests', function(){
+
+    const request = chai.request;
+
+    describe('Create issue', function(){        
+
+        describe('Valid params', function() {
+
+            let issue;
+            before((done) => {
+                issue = {
```

Comment:
We gonna start being picky now 😄 That being said... You have named this variable `issue`. One could assume that the object like this represents issue. Which is only partially true. This is more like issue params/attributes. So variables like this should be named `issueParams` or `issueAttributes` or something third depicting better what this is.